docs/core/tools/dotnet-format.md

```diff
@@ -0,0 +1,146 @@
+
+* **`--verify-no-changes`**
+
+  Verifies that no formatting changes would be performed. Terminates with a non zero exit code if any files would have been formatted.
+
+* **`--include <INCLUDE>`**
+
+  A space-separated list of relative file or folder paths to include in formatting. All files in the solution or project are formatted if empty.
+
+* **`--exclude <EXCLUDE>`**
+
+  A space-separated list of relative file or folder paths to exclude from formatting.  The default is none.
+
+* **`--include-generated`**
+
+  Formats files generated by the SDK.
+
+* **`-v|--verbosity <LEVEL>`**
+
+  Sets the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`. Default value is `m[inimal]`.
+
+* **`--binarylog <BINARY-LOG-PATH>`**
+
+  Logs all project or solution load information to a binary log file.
+
+* **`--report <REPORT-PATH>`**
+
+  Produces a JSON report in the directory specified by `<REPORT_PATH>`.
+
+* **`-h|--help`**
+
+  Shows help and usage information
+
+## Subcommands
+
+### Whitespace
+
+`dotnet format whitespace` - Formats code to match `editorconfig` settings for whitespace.
+
+#### Description
+
+The `dotnet format whitespace` subcommand will only run formatting rules associated with whitespace formatting. For a complete list of possible formatting options that you can specify in your *.editorconfig* file see the [whitespace formatting documentation](/visualstudio/ide/reference/options-text-editor-csharp-formatting).
+
+### Style
+
+`dotnet format style` - Formats code to match EditorConfig settings for code style.
+
+#### Description
+
+The `dotnet format style` subcommand will only run formatting rule associated with code style formatting. For a complete list of possible formatting options that you can specify in your `editorconfig` file see the [code style documentation](../../fundamentals/code-analysis/style-rules/language-rules.md).
+
+#### Options
+
+* **`--severity`**
+
+  The minimum severity of diagnostics to fix. Allowed values are `info`, `warn`, and `error`. The default value is `warn`
+
+### Analyzers
+
+`dotnet format analyzers` - Formats code to match `editorconfig` settings for analyzers.
+
+#### Description
+
+The `dotnet format analyzers` subcommand will only run formatting rule associated with analyzers. For a list of possible analyzer rules that you can specify in your `editorconfig` file see the  [list of ids for code-analysis quality rules](../../fundamentals/code-analysis/quality-rules/index.md).
+
+##### Options
+
+* **`--diagnostics <DIAGNOSTICS>`**
+
+  A space separated list of diagnostic ids to use as a filter when fixing code style or 3rd party issues. Default value is whichever ids are listed in the `editorconfig` file. For a list of built-in analyzer rule ids that you can specify see the list of ids see the [documentation of code-analysis quality rules](../../fundamentals/code-analysis/quality-rules/index.md).
```

Comment:
Did this mean to say "code quality" instead of "code style"?

```suggestion
  A space separated list of diagnostic ids to use as a filter when fixing code quality or 3rd party issues. Default value is whichever ids are listed in the `editorconfig` file. For a list of built-in analyzer rule ids that you can specify see the list of ids see the [documentation of code-analysis quality rules](../../fundamentals/code-analysis/quality-rules/index.md).
```